### DISCUSSION TITLE
Provide some basic troubleshooting

### DISCUSSION THREAD

**Discussion by cloudxabide** [chat-cli/chat-cli#64] [Ideas]
Generic/basic suggestions:  
Ensure you are using the AWS profile (and correct region) you intended by reviewing `aws configure list`

More complex suggestions:
if you receive an error such as "AccessDeniedException: You don't have access to the model with the specified model ID.", you should ensure you have explicitly called the model you intended, and if you have, then should review CloudTrail logs.

**Comment by micahwalter** [comment]
Thanks @cloudxabide  - I moved this to a discussion topic.. 

Can you elaborate on what you are thinking here? How would this work via the CLI tool, or are you just thinking about documentation?

For the 2nd one, I have played with the idea of letting users tail recent CloudTrail logs, but it comes with some complexity in setup/config and filtering what is relevant.

-m

**Reply by cloudxabide** [reply]
I was thinking more of a documentation page for "common" challenges one may encounter. 

I imagine that people from varying skill levels may find the chat-cli repo and may encounter some nuanced issues.  (I, personally, have forgot to configure a default region for individual AWS profiles.  
Another example is when a new model was recently released (nova) and I had not thought to go enable it.  I believe for one of the tasks I was attempting, I received a permission denied error.  (I believe for whatever task I was attempting, the command defaulted to nova since I had not explicitly called a model).

As I was answering the question about whether it would be a documentation or CLI option - it did seem like a cool idea if the non-success return code was "picked up" and reviewed by Bedrock to provide suggestions as to how to remedy the issue.

**Reply by micahwalter** [reply]
Nice. I am starting a more in depth docs site at https://docs.chat-cli.chat -- It's part of the same repo under the `/docs` folder (if you are interested to contribute to that, please do!).
